dev/Dockerfile.base

```diff
@@ -47,6 +48,13 @@ RUN set -ex; \
     && mkdir --mode=2775 /app \
     && chown ${USER_NAME}:${USER_GROUP} /app
 
+# Install entr so we can have reload on worker processes
+RUN git clone https://github.com/eradman/entr && \
+    cd entr && \
+    cp Makefile.linux Makefile && \
+    make && \
+    make install
```

Comment:
`entr` is available on fedore repos but not for our redhat base image, so decided to build it from git.